processor/k8sattributesprocessor/e2e_test.go

```diff
@@ -253,7 +242,7 @@ func TestE2E(t *testing.T) {
 			name:     "metrics-daemonset",
 			dataType: component.DataTypeMetrics,
 			service:  "test-metrics-daemonset",
-			attrs: map[string]*expectedValue{
+			attrs: map[string]*k8stest.ExpectedValue{
```

Comment:
These changes still need to be reverted right?

Reply:
Missed that, thanks!